auslib/web/admin/views/problem.py

```diff
@@ -0,0 +1,44 @@
+import json
+from flask import Response
+
+
+def problem(status, title, detail, type=None, instance=None, headers=None, ext=None):
```

Comment:
Very strange. Maybe something to follow-up on if there ends up being extra time in the end. This is certainly fine for now though.